magic-links/24hour/24hour.js

```diff
@@ -17,6 +17,10 @@ let escapeObject = (obj) => {
   }))
 }
 
+var displayMailChimpStatus = function (data) {
+  console.log(data)
```

Comment:
?

Reply:
it's a debug/getting the response strings out of the mailchimps api response so we could if we wanted to echo out errors/warnings etc. from the mailchimp signup request.

we can get rid of it if we don't want to do that, but hadn't landed one way or the other as wanted to chat it through with you first

Reply:
if you watch the console while signing up you'll see them